04_context.js

```diff
@@ -0,0 +1,6 @@
+function calc() {
+  return this.a + this.b;
+}
+
+let calcWithContext = calc.bind({a: 3, b: 4});
```

Comment:
Do it without creating additional wrapper please. Call function directly

Reply:
done